store/__tests__/playbackManager.spec.ts

```diff
@@ -52,721 +52,745 @@ beforeEach(() => {
+    store.replaceState({
+      ...defaultState(),
+      currentItemIndex: 5
+    });
 
-  expect(store.state.currentItemIndex).toBe(5);
-});
+    store.dispatch('setCurrentIndex', { index: 5 });
 
-test('When setVolume is is called, the volumeLevel is updated. Case B', () => {
-  store.replaceState({
-    ...defaultState(),
-    currentItemIndex: 5
+    expect(store.state.currentItemIndex).toBe(5);
   });
 
-  store.dispatch('setCurrentIndex', { index: 6 });
+  it('sets the current item index when setCurrentIndex is dispatched with a new index', () => {
+    store.replaceState({
+      ...defaultState(),
+      currentItemIndex: 5
+    });
 
-  expect(store.state.currentItemIndex).toBe(6);
-});
+    store.dispatch('setCurrentIndex', { index: 6 });
 
-test('When setCurrentTime is is called, the currentTime is updated.', () => {
-  store.replaceState({
-    ...defaultState()
+    expect(store.state.currentItemIndex).toBe(6);
   });
 
-  store.dispatch('setCurrentTime', { time: 12.345 });
+  it('sets the current time when when setCurrentTime is dispatched', () => {
+    store.replaceState({
+      ...defaultState()
+    });
 
-  expect(store.state.currentTime).toBe(12.345);
-});
+    store.dispatch('setCurrentTime', { time: 12.345 });
 
-test('When changeCurrentTime is is called, the currentTime is updated.', () => {
-  store.replaceState({
-    ...defaultState()
+    expect(store.state.currentTime).toBe(12.345);
   });
 
-  store.dispatch('changeCurrentTime', { time: 12.345 });
+  it('sets the current time when when changeCurrentTime is dispatched', () => {
+    store.replaceState({
+      ...defaultState()
+    });
 
-  expect(store.state.currentTime).toBe(12.345);
-});
+    store.dispatch('changeCurrentTime', { time: 12.345 });
 
-test('When skipForward is is called, the currentTime is updated.', () => {
-  store.replaceState({
-    ...defaultState(),
-    currentTime: 10
+    expect(store.state.currentTime).toBe(12.345);
   });
 
-  store.dispatch('skipForward');
+  it('updates the current time when skipForward is dispatched', () => {
+    store.replaceState({
+      ...defaultState(),
+      currentTime: 10
+    });
 
-  expect(store.state.currentTime).toBe(25);
-});
+    store.dispatch('skipForward');
 
-test('When skipBackward is is called, the currentTime is updated.', () => {
-  store.replaceState({
-    ...defaultState(),
-    currentTime: 10
+    expect(store.state.currentTime).toBe(25);
   });
 
-  store.dispatch('skipBackward');
+  it('resets the current time when skipBackward is dispatched when at 10 seconds or less', () => {
+    store.replaceState({
+      ...defaultState(),
+      currentTime: 10
+    });
 
-  expect(store.state.currentTime).toBe(0);
-});
+    store.dispatch('skipBackward');
 
-test('When skipBackward is is called, the currentTime is updated. Case B', () => {
-  store.replaceState({
-    ...defaultState(),
-    currentTime: 25
+    expect(store.state.currentTime).toBe(0);
```

Comment:
Defaultstate ?

Reply:
It has to be 0, the default state is null.